src/ResourceManagement/DataLake.Store/Microsoft.Azure.Management.DataLake.Store/Customizations/FileSystemOperationsExtensions.Customizations.cs

```diff
@@ -59,6 +59,5 @@ public static bool PathExists(this IFileSystemOperations operations, string acco
                 return _result.Body;
             }
         }
-
```

Comment:
Stylecop was complaining about a blank line with no code after it. Is it important to add it back?